_posts/en/newsletters/2023-09-13-newsletter.md

```diff
@@ -0,0 +1,186 @@
+---
+title: 'Bitcoin Optech Newsletter #268'
+permalink: /en/newsletters/2023/09/13/
+name: 2023-09-13-newsletter
+slug: 2023-09-13-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter FIXME
+
+## News
+
+- **Specifications for taproot assets:** Olaoluwa Osuntokun posted
+  separately to the Bitcoin-Dev and Lightning-Dev mailing lists about
+  the _Taproot Assets_ [client-side validation protocol][topic client-side
+  validation].  To the Bitcoin-Dev mailing list, he [announced][osuntokun
+  bips] seven draft BIPs---one more than in the initial announcement of
+  the protocol, then under the name _Taro_ (see [Newsletter
+  #195][news195 taro]).  To the Lightning-Dev mailing list, he
+  [announced][osuntokun blip post] a [draft BLIP][osuntokun blip] for
+  spending and receiving taproot assets using LN, with the protocol
+  based on the experimental "simple taproot channels" feature planned to
+  be released in LND 0.17.0-beta.
+
+    Note that, despite its name, Taproot Assets is not part of the
+    Bitcoin Protocol and does not change the consensus protocol in any
+    way.  It uses existing capabilities to provide new features for
+    users that opt-in to its client protocol.
+
+    None of the specifications had received any discussion on the
+    mailing list as of this writing.
+
+- **LN messaging changes for PTLCs:** as the first LN implementation
+  with experimental support for channels using [P2TR][topic taproot]
+  and [MuSig2][topic musig] is expected to be released soon, Greg
+  Sanders [posted][sanders post] to the Lightning-Dev mailing list a
+  [summary][sanders ptlc] of several different previously-discussed
+  changes to LN messages to allow them to support sending payments with
+  [PTLCs][topic ptlc] instead of [HTLCs][topic htlc].  For most
+  approaches, the changes to messages do not seem large or invasive,
+  but we note that most implementations will probably continue using one
+  set of messages for handling legacy HTLC forwarding while also
+  offering upgraded messages to support PTLC forwarding, creating two
+  different paths that will need to be maintained concurrently until
+  HTLCs are phased out.  If some implementations add experimental PTLCs
+  support before the messages are standardized, then implementations
+  might even be required to support three or more different protocols
+  simultaneously, to the disadvantage of all.
+
+    Sander's summary had not received any comments as of this writing.
```

Comment:
```suggestion
    Sander's summary has not received any comments as of this writing.
```
This comment is going to set the nittyness record, but the verb "had" slightly implies that it did receive comments _after_ the time of this writing.